subt/visual_odometry.py

```diff
@@ -1043,7 +1043,7 @@ def __maybe_update_odometry_and_visualize(self):
             return True
 
         if (self.grey_img is None or self.depth is None
-                or abs(self.depth_time - self.img_time) > 1e-2):
+                or abs(self.depth_time - self.img_time) > 0.15):
```

Comment:
This change is not exciting. We need to find a better solution for "bundling" matching depth and image data. (In a separate change.)